atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbRemoteException.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.http;
+
+import com.palantir.remoting2.errors.RemoteException;
+
+/**
+ * An AtlasDbRemoteException is a wrapper around a http-remoting RemoteException.
+ *
+ * This is necessary because we shadow http-remoting to avoid dependency conflicts with AtlasDB clients, which may
+ * be using different versions of http-remoting.
+ */
+public class AtlasDbRemoteException extends RuntimeException {
+    private static final long serialVersionUID = 1L;
+
+    private final RemoteException remoteException;
```

Comment:
We could, though my concern was that we would need to add a wrapper for `SerializableStackTraceElement` as well. Discussed offline - probably worthwhile to have clients expect the same / a similar API.